backend/apps/startups/views.py

```diff
@@ -1,3 +1,8 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
+from .models import StartupProfile
+from .serializers import StartupPublicProfileSerializer
 
-# Create your views here.
+class StartupPublicProfileViewSet(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
+    queryset = StartupProfile.objects.all()
+    serializer_class = StartupPublicProfileSerializer
+    lookup_field = 'id'
```

Comment:
lookup_field = 'id'
DRF uses the pk field by default.
If the StartupProfile model has a regular id field = models.AutoField(primary_key=True) - there is no point in changing it.
It is recommended to simply remove this line to leave the default behavior: